src/main/java/jp/t2v/xanadu/ops/TakeWhileSpliterator.java

```diff
@@ -0,0 +1,34 @@
+package jp.t2v.xanadu.ops;
+
+
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+class TakeWhileSpliterator<T> extends Spliterators.AbstractSpliterator<T> {
```

Comment:
I want to put only `FooOps` under `ops` package directly.
So moving this class to StreamOps private nested class is good.
What do you think?